res/values-tl/strings.xml

```diff
@@ -1012,7 +1012,7 @@
     <string name="POSITION_CANCEL_BUTTON">POSISYON SA CANCEL BUTTON</string>
     <string name="INVISIBILITY">INVISIBILITY</string>
     <string name="Set_Skin">I-Set Skin</string>
-    <string name="Primary">Primary</string>
+    <string name="Primary">Pangunahin</string>
     <string name="Secondary">Secondary</string>
```

Comment:
Isn't this supposed to be translated too?

```suggestion
    <string name="Secondary">Pangalawang</string>

Reply:
No

Reply:
Not "Pangalawang" its "Pangalawa"